charts/private-action-runner/README.md.gotmpl

```diff
@@ -1,9 +1,18 @@
 # Datadog Private Action Runner
 
-![Version: 0.6.0](https://img.shields.io/badge/Version-0.6.0-informational?style=flat-square) ![AppVersion: v0.0.1-alpha27](https://img.shields.io/badge/AppVersion-v0.0.1--alpha27-informational?style=flat-square)
+![Version: 0.7.0](https://img.shields.io/badge/Version-0.7.0-informational?style=flat-square) ![AppVersion: v0.0.1-alpha27](https://img.shields.io/badge/AppVersion-v0.0.1--alpha27-informational?style=flat-square)
```

Comment:
do you know what this file is for? just seems like a clone of the readme to me

Reply:
yeah you are supposed to edit this file and `values.yaml` only then run `.github/helm-docs.sh` to make changes to the README.
https://github.com/DataDog/helm-charts/blob/b842ff457b184841dd140d9ab6726eb915b0d7ab/CONTRIBUTING.md#how-to-update-a-readme-file

Reply:
ah i see